web/src/components/DandisetsPage.vue

```diff
@@ -33,7 +41,7 @@
       class="
         mx-12
         my-12"
-      :dandisets="dandisets"
+      :dandisets="dandisets || []"
```

Comment:
I would prefer a sensible default be provided rather than having to do this edge case handling here:
```
asyncComputed: {
  dandisets: {
    get() {
     ...
      return ...;
    },
    default: [],
  },
}
```
It's more verbose, but it keeps logic out of the template and presents a cleaner API between the script and the template.

Reply:
I agree, that's a better way to handle it. I didn't know you could do that.

Reply:
I actually don't think this'll happen anymore, because of the other change regarding storing the request as a async computed prop, but I like that you can do this.

Reply:
In either case, I think the ` || []` can be removed safely now